de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/presenter/callbacks/SaveTemplateError.java

```diff
@@ -0,0 +1,8 @@
+package org.iplantc.de.admin.desktop.client.metadata.presenter.callbacks;
+
+import jsinterop.annotations.JsFunction;
+
+@JsFunction
+public interface SaveTemplateError {
```

Comment:
I see you haven't used `@supperssWarnings('unusable-by-js')`. You didn't get any warnings ?